toponetx/datasets/mesh.py

```diff
@@ -36,14 +35,14 @@ def stanford_bunny(complex_type="simplicial complex"):
     ValueError
         If complex_type is not one of the supported values.
     """
-    if complex_type == "simplicial complex":
+    if complex_type == "simplicial":
         cpx = SimplicialComplex.load_mesh(DIR + "/bunny.obj")
         return cpx
-    elif complex_type == "cell complex":
+    elif complex_type == "cell":
         cpx = CellComplex.load_mesh(DIR + "/bunny.obj")
         return cpx
     else:
```

Comment:
No need for else because the above two if-statements end with "return". Remove.

Reply:
what if the complex is not of the types the user input? that is the purpose of the third one, no?